packages/local/XAdmin/ui/src/hooks/useProducers.ts

```diff
@@ -4,11 +4,21 @@ import { type Producer, getProducers } from "@/lib/graphql/getProducers";
 import { queryKeys } from "@/lib/queryKeys";
 
 import { useConfig } from "./useConfig";
+import { useStatuses } from "./useStatuses";
 
 export const useProducers = () => {
+    const { data: status, error: statusError } = useStatuses();
     return useQuery<Producer[]>({
         queryKey: queryKeys.producers,
         queryFn: async () => {
+            if (
+                ["startup", "slow", "needgenesis"].includes(
```

Comment:
`Slow` doesn't seem right here, does it? And I assume startup is a status while starting up?